readthedocs/doc_builder/python_environments.py

```diff
@@ -472,8 +543,19 @@ def install_core_requirements(self):
             pip_requirements.append('mkdocs')
         else:
             pip_requirements.append('readthedocs-sphinx-ext')
-            requirements.extend(['sphinx', 'sphinx_rtd_theme'])
+            conda_requirements.extend(['sphinx', 'sphinx_rtd_theme'])
```

Comment:
I believe we explicitly don't add versions here just to confirm that they are installed without specifying a version or having them be upgraded. Has this changed? I don't know how this logic would be upgrading users versions.

Reply:
Once the environment is created with `conda env create` if the user pin `sphinx==1.6` it will be upgraded (in the current production code) in the step `conda install sphinx` --even if we don't specify any version.

There was an attempt to add an attribute in `conda` itself called `--satisfied-skip-solve` but does not work as we need unfortunately :( . See #5631